.github/workflows/check_code.yml

```
@@ -0,0 +1,15 @@
+name: Flake8 check
+on: [pull_request]
+jobs:
+    flake8 unused imports:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Install flake8
+              run: pip install flake8
+            - name: Checking executables for unused imports
+              run: flake8 --select=F401 `find bin -type f`
```

From the description it sounds like you wanted this:
```suggestion
              run: flake8 --select=F401 `find bin -type f -name pycbc_*`
```